change/@ni-nimble-angular-69e47a5c-036b-401c-b2c7-5c5cbc0ea07f.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Doesn't look like there are any Angular or Blazor changes anymore so those changefiles can be deleted.